src/chapters/08/main.tex

```diff
@@ -115,62 +115,67 @@ \section{Theory}\label{sec:theory}
 \end{figure}
 
 Expression~\ref{eqn:objective_paint} corresponds to the total profit, which is
-the expression we are trying to maximise. As a line in the 2-dimensional space,
+the value to be maximised. As a line in 2-dimensional space,
 this expression fixes its gradient, but its value determines the size of the
-$y$-intercept. Therefore optimising this function corresponds to pushing a line
+\(y\)-intercept. Therefore optimising this function corresponds to pushing a line
 with that gradient to its furthest extreme within the feasible region,
```

Comment:
The definition of `feasible region` was removed above, and it's interpretation as a convex region bounded by the constraints. Should we define that here?

Reply:
`The shaded region in the Figure ... shows the pairs of values of A and B which are feasible, that is: they satisfy the constraints.`